aptos-move/framework/src/natives/aggregator_natives/aggregator.rs

```diff
@@ -62,6 +62,17 @@ fn native_add(
     Ok(smallvec![])
 }
 
+fn native_try_add(
+    gas_params: &AddGasParameters,
+    context: &mut SafeNativeContext,
+    _ty_args: Vec<Type>,
+    args: VecDeque<Value>,
+) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    Ok(smallvec![Value::bool(
+        try_add(gas_params, context, _ty_args, args).is_ok()
```

Comment:
hmm, so the false doesn't only indicate an overflow, but could be any potential errors happening inside? I'd prefer to separate the overflow case out while still propagate other errors up